tests/Oryx.Integration.Tests/LocalDockerTests/DotNetCoreEndToEndTests.cs

```diff
@@ -567,7 +567,8 @@ public async Task CanRunApp_WhenTheRootDirectoryHasStartupDllFile()
             var appOutputDir = appOutputDirVolume.ContainerDir;
             var portMapping = $"{HostPort}:{ContainerPort}";
             var buildScript = new ShellScriptBuilder()
-                .AddCommand($"oryx build {appDir} -o {appOutputDir} -l dotnet --language-version {dotnetcoreVersion}")
+                .AddCommand($"oryx build {appDir} -i /tmp/int -o /tmp/out -l dotnet --language-version {dotnetcoreVersion}")
```

Comment:
FYI: tested this test alone as part of nightly build and it works fine: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2546684